specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/common/v1/definitions.json

```diff
@@ -193,6 +193,22 @@
       "minLength": 1,
       "maxLength": 90,
       "x-ms-parameter-location": "method"
+    },
+    "LocationParameter": {
+      "name": "location",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The region to which the request is sent. You can find out which regions Azure Stream Analytics is supported in here: https://azure.microsoft.com/en-us/regions/",
+      "x-ms-parameter-location": "method"
+    },
+    "OperationIdParameter": {
+      "name": "operationId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The operation ID returned by the initial query testing request.",
```

Comment:
Operation Id is a generic concept for any async api. It is not specific to query testing.